static/css/restyle.less

```diff
@@ -0,0 +1,853 @@
+// Colors
+@primary-blue: rgba( 12, 153, 213, 1);
+@primary-link: #6AC3E2;
+
+#tabzilla:before {
+    background: none;
+}
+
+body {
+    font-family: "Open Sans",X-LocaleSpecific,sans-serif !important;
```

Comment:
Spaces after the commas would make this a bit easier to scan, I think.

Shame about the `!important`, but I get why it's there.